### PR TITLE
docs: update SR docs

### PR DESCRIPTION
**Pull request by YuLiu98**

### CODE REVIEW

File: docs/advanced/elec_properties/hs_matrix.md [hunk 1]
```diff
@@ -61,6 +61,11 @@ We also offer the option of only calculating the overlap matrix without running
 
 A file named `SR.csr` will be generated in the working directory, which contains the overlap matrix.
 
+> When `nspin` is set to 1 or 2, the dimension of the overlap matrix is nlocal $\times$ nlocal, where nlocal is the total number of numerical atomic orbitals. 
+These numerical atomic orbitals are ordered from outer to inner loop as atom, angular  quantum number $l$, zeta, and magnetic quantum number $m$.
```
Comment (mohanchen) [inline docs/advanced/elec_properties/hs_matrix.md]: one extra space is foud after 'angular'

Comment (mohanchen) [reply docs/advanced/elec_properties/hs_matrix.md]: please explain zeta

File: docs/advanced/elec_properties/hs_matrix.md [hunk 2]
```diff
@@ -61,6 +61,11 @@ We also offer the option of only calculating the overlap matrix without running
 
 A file named `SR.csr` will be generated in the working directory, which contains the overlap matrix.
 
+> When `nspin` is set to 1 or 2, the dimension of the overlap matrix is nlocal $\times$ nlocal, where nlocal is the total number of numerical atomic orbitals. 
+These numerical atomic orbitals are ordered from outer to inner loop as atom, angular  quantum number $l$, zeta, and magnetic quantum number $m$. 
+When `nspin` is set to 4, the dimension of the overlap matrix is (2 $\times$ nlocal) $\times$ (2 $\times$ nlocal). In this case, the numerical atomic orbitals are ordered from outer to inner loop as atom, angular  quantum number $l$, zeta, magnetic quantum number $m$, and npol (npol ranges from 0 to 1).
```
Comment (mohanchen) [inline docs/advanced/elec_properties/hs_matrix.md]: please explain npol: index of spin

File: docs/advanced/elec_properties/position_matrix.md [hunk 3]
```diff
@@ -18,4 +18,7 @@ Each file or each section of the appended file starts with "STEP: " followed by
...
 
-In molecular dynamics (MD) calculations, if [out_app_flag](../input_files/input-main.md#out_app_flag) is set to true, then `data-rR-tr` is written in an append manner. Otherwise, output files will be put in a separate directory, `matrix`, and named as `$x`_data-rR-tr, where `$x` is the number of MD step. In addition, The output frequency is controlled by [out_interval](../input_files/input-main.md#out_interval). For example, if we are running a 10-step MD with out_interval = 3, then `$x` will be 0, 3, 6, and 9.
\ No newline at end of file
+In molecular dynamics (MD) calculations, if [out_app_flag](../input_files/input-main.md#out_app_flag) is set to true, then `data-rR-tr` is written in an append manner. Otherwise, output files will be put in a separate directory, `matrix`, and named as `$x`_data-rR-tr, where `$x` is the number of MD step. In addition, The output frequency is controlled by [out_interval](../input_files/input-main.md#out_interval). For example, if we are running a 10-step MD with out_interval = 3, then `$x` will be 0, 3, 6, and 9.
```
Comment (mohanchen) [inline docs/advanced/elec_properties/position_matrix.md]: The should be 'the' (after "in addition,")

File: docs/advanced/elec_properties/position_matrix.md [hunk 4]
```diff
@@ -18,4 +18,7 @@ Each file or each section of the appended file starts with "STEP: " followed by
...
+
+## get_S
+We also offer the option of only calculating the position matrix without running SCF. For that purpose, in `INPUT` file we need to set the value keyword [calculation](../input_files/input-main.md#calculation) to be `get_S`.
```
Comment (mohanchen) [inline docs/advanced/elec_properties/position_matrix.md]: Besides setting 'get_S', the 'PARAM.inp.out_mat_r' (which input parameter?) should be true

Comment (YuLiu98) [reply docs/advanced/elec_properties/position_matrix.md]: The related input parameter is mentioned in the above content of the doc.

**Review by mohanchen — APPROVED** [review]

**Pull request merged**